### PR TITLE
Set up CircleCI Deployment

### PR DESCRIPTION
**Pull request by dongbohu**

### CODE REVIEW

**Comment by dhimmel** [comment]
So this deployment script has the server pull the latest master git commit. But it doesn't have the ability to update the environment right? For example, if we modify `environment.yml`, we'll still have to login to the backend server and do manual configuration?

**Comment by dongbohu** [comment]
True. But isn't the conda environment being used only for development?

**Comment by dhimmel** [comment]
> But isn't the conda environment being used only for development?

You tell me? We need a way of specifying exact package versions for both development and deployment. Ideally, the development environment is very similar if not identical to the deployment environment.

**Comment by dongbohu** [comment]
We could add the following line before the `sudo` command in `deploy.sh`:
```
conda update --name hetmech-backend --file ~/hetmech-backend/environment.yml
```
This will update the conda env. If we don't modify `environment.yml` that often, there is no big deal. I don't mind doing it manually.

**Comment by dhimmel** [comment]
I like the idea of using the `conda udpate` command. Will the CircleCI build logs include the output from this SSH session? It would be helpful to see the output of the conda update command because sometimes it's a bit wonky.

File: dj_hetmech/settings.py [hunk 1]
```diff
@@ -45,8 +45,10 @@
     'django.contrib.staticfiles',
     'django_extensions',
     'rest_framework',
+    'django_filters',
     'corsheaders',
     'dj_hetmech_app',
+
```
Comment (dhimmel) [inline dj_hetmech/settings.py]: This blank line should go?

Comment (dongbohu) [reply dj_hetmech/settings.py]: Done

**Review by dhimmel — APPROVED** [review]

**Pull request merged**